common/changes/@itwin/tree-widget-react/Fix_show_all_button_in_model_tree_2023-03-06-09-14.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/tree-widget-react",
+      "comment": "tree-widget: ensure that the \"show all\" and \"hide all\" buttons affect all iModel content",
```

Comment:
The message will show up in the CHANGELOG of `tree-widget-react` package - no need to prefix this with `tree-widget`. Instead, I think we should prefix with the name of component that gets affected by the change.